src/engine-actions/close-channels.js

```diff
@@ -0,0 +1,83 @@
+const {
+  listChannels,
+  closeChannel,
+  listPendingChannels
+} = require('../lnd-actions')
+/**
+ * Closes a channel given the channel point
+ *
+ * @param {String} paymentChannelNetworkAddress
+ * @param {String} fundingAmount - int64 string
+ * @returns {Promise<void>} returns void on success
+ */
+async function closeChannels ({ force = false, pending = false } = {}) {
+  const { channels = [] } = await listChannels({ client: this.client })
+  const activeChannels = channels.filter(c => c.active)
+
+  let inactiveChannels = []
+  if (force === true) {
+    inactiveChannels = channels.filter(c => c.active === false)
+  }
+
+  let pendingChannels = []
+  if (pending === true) {
+    const { pendingOpenChannels = [] } = await listPendingChannels({ client: this.client })
+    pendingChannels = pendingOpenChannels
+  }
```

Comment:
At this point in the code, if all `activeChannels`, `inactiveChannels` or `pendingChannels` is of length 0, then we should return early and log.